### ISSUE TITLE
Invalid nesting of header within header

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#570]
Area of Site: Homepage
URL: https://hcommons-test.org/

**Description**
There is a `header` element nested within another `header` element. This is considered invalid HTML as header elements cannot be descendants of other header elements. It also presents accessibility issues related to the page's landmarks. 

**Solution**
Remove the parent `header` element.
